runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -94,6 +94,14 @@ var spec = drivers.Spec{
 			Description: "Name of the ClickHouse database to connect to",
 			Placeholder: "default",
 		},
+		{
```

Comment:
I've only made minor changes to the Go files. Any idea as to why this [Test Go code](https://github.com/rilldata/rill/actions/runs/16030574210/job/45229819479?pr=7450) is failing? @AdityaHegde

Reply:
nvm, I see it happening in https://github.com/rilldata/rill/actions/runs/16017688761/job/45187392578?pr=7506 too